cpp/source/deal.II/geometry.h

```diff
@@ -72,6 +81,13 @@ class Geometry
     return _materials;
   }
 
+  void set_materials(
+      std::shared_ptr<std::unordered_map<
+          std::string, std::vector<dealii::types::material_id>>> materials)
+  {
+    _materials = materials;
+  }
+
```

Comment:
I don't think that it is necessary now. We don't use the other boundary ids and we have never needed to use multiple boundary ids for one boundary. If we need it later, we can add it at that time.